src/common/asset-handler.js

```diff
@@ -793,6 +793,7 @@ var assetHandler = (function() {
                             },
                             error: hardWay
                         });
+                    // eslint-disable-next-line no-unused-vars
```

Comment:
I think these try/catch statements which define a variable we don't use, we can remove the unused variable, like this:
```
try {
    // ...
} catch {
    // ...
}
```
Which is a little better than an eslint ignore.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch